docs/other/model-in-memory.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: model_in_memory
+title: Creating AMF model in memory
+---
+:::info
+The functionality expressed in this documentation requires familiarity with the
+[amf model](https://github.com/aml-org/amf/blob/develop/documentation/model.md).
+:::
+
+AMF is most commonly used to parse an already existing API,
+but it can also be used to create an API in a fluent and descriptive way.
+
+Some use cases for creating a WebApi model in memory include:
```

Comment:
When you create an API [or Web API???] model in memory, you can use AMF to:

- Edit an already parsed API
- Power a visual API designer
- Create programmatic API tests